doc/installation/index.rst

```diff
@@ -121,17 +131,17 @@ GnuTLS **(>= 2.8)**         gnutls-devel               Development files for gnu
 
 #. Install Centreon repository
 
-   You need to install Centreon Entreprise Server (CES) repos file as
+   You need to install Centreon Open Sources (COS) repository file as
    explained :ref:`user_installation_packages_prerequisites` to use some
    specific package version.
 
-#. Install RRDTool ::
+#. Install cmake and lua-devel ::
 
-   $ yum install rrdtool-devel
+  $ yum install cmake lua-devel
 
-#. Install cmake ::
+#. Install RRDTool ::
```

Comment:
If you look at the rendered page, you will notice that the enumeration starts from the beginning instead of continuing with number 4.